hoomd/write/table.py

```diff
@@ -38,6 +39,12 @@ def __subclasshook__(cls, C):
             return NotImplemented
 
 
+def _writable(fh):
```

Comment:
Indicate that this is a "check" function in its name.
```suggestion
def _ensure_writable(fh):
```